stock_request/tests/__init__.py

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
```

Comment:
[The first line # -*- coding: utf-8 -*- is not needed anymore in Python 3 if you save it in that encoding.](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0)